eng/Version.Details.xml

```diff
@@ -7,9 +7,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>620462dd065490620c03a80cea8e251004f6bf21</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.9.0-4.20631.6">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.9.0-4.21076.10">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>c34bb049f1f74256f109794509cd9f0f1c23faea</Sha>
```

Comment:
It looks like only the compilers toolset is updated here, not the arcade version.

Reply:
I just cherry-pick the change from here https://github.com/dotnet/roslyn/pull/50828
Do you feel the update of compiler toolset is not needed?

Reply:
I think it's OK to update the toolset. I mentioned this to make sure of whether it's expected to have an arcade update in this PR.